src/js/xAxisTimeSeries.js

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Mix-in grade for time-series x axis
+
+    fluid.defaults("floe.chartAuthoring.xAxisTimeSeries", {
+        model: {
+            dataSet: []
+        },
+        axisOptions: {
+            numberOfXAxisTicks: 6
```

Comment:
Looks like there's a case for at least 3 grades - plain x-axis, plain y-axis, x-axis time series, and the "base" axis grade which it seems right now isn't a base for any of these since it is "stateless". If any of these grades depends for "typical use" on any of the others, they should be mentioned in leading comments